neqo-http3/src/connection_client.rs

```diff
@@ -692,6 +692,11 @@ impl Http3Client {
         Ok(())
     }
 
+    pub fn change_max_stream_data(&mut self, stream_id: u64, max_data: u64) -> Res<()> {
+        self.conn.change_max_stream_data(stream_id, max_data)?;
+        Ok(())
+    }
```

Comment:
Is this changing the value of MAX_DATA or the size of the receive window?  I think that it's the latter, in which case this should be `c.set_receive_window(stream_id, window)`

Reply:
> Check the title of this PR. This is less about refactoring and more about setting the size of the receive buffer.
> 

I forgot to note that this is based on #1125 so the first change set will land in that PR and this one is only about streams flow control.